docs/algorithms.md

```diff
@@ -151,12 +212,44 @@ To learn more about Integrated Gradients, visit the following resources:
 - [Original paper](https://arxiv.org/abs/1703.01365)
 
 ### Neuron Guided Backpropagation
-Neuron guided backpropagation is the analog of guided backpropagation for a particular neuron. It computes the gradient of the target neuron with respect the input, but backpropagation of ReLU functions is overridden so that only non-negative gradients are backpropagated (any negative gradients are set to 0). Guided backpropagation was proposed in the context of an all-convolutional network and is generally used for neurons in convolutional networks, although the approach can be applied generically.
+Neuron guided backpropagation is the analog of guided backpropagation for a particular neuron. It computes the gradient of the target neuron with respect the input, but backpropagation of ReLU functions is overridden so that only non-negative gradients are backpropagated (applying ReLU to the input gradient) Guided backpropagation was proposed in the context of an all-convolutional network and is generally used for neurons in convolutional networks, although the approach can be applied generically.
```

Comment:
I think that `Neuron Guided Backpropagation` and  `Neuron Deconvolution` can be combined similar to the `Guided Backpropagation and Deconvolution`. We can also refer here to `Guided Backpropagation and Deconvolution`